app/design/frontend/Magento/luma/Magento_Theme/layout/default_head_blocks.xml

```diff
@@ -8,10 +8,5 @@
 <page xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="urn:magento:framework:View/Layout/etc/page_configuration.xsd">
     <head>
         <meta name="viewport" content="width=device-width, initial-scale=1"/>
-        <font src="fonts/opensans/light/opensans-300.woff2"/>
-        <font src="fonts/opensans/regular/opensans-400.woff2"/>
-        <font src="fonts/opensans/semibold/opensans-600.woff2"/>
-        <font src="fonts/opensans/bold/opensans-700.woff2"/>
-        <font src="fonts/Luma-Icons.woff2"/>
```

Comment:
These lines might not be required but they are actually added for a good reason. This will make sure the fonts are preloaded, which is what you want because you want your custom fonts to be shown as soon as possible.

This will compile to something like this:
```
<link rel="preload" as="font" crossorigin="anonymous" href="https://domain.com/path-to-fonts/fonts/opensans/light/opensans-300.woff2" />
```

Preloading these fonts ensures they are available earlier and are less likely to block the page's render, improving performance. See https://developer.mozilla.org/en-US/docs/Web/HTML/Preloading_content

Reply:
Hello @krzksz, Can you perhaps take another look at this PR? I'm not sure the changes in this PR are code improvements, see my comments.

Reply:
fonts already import in parent blank so we don't need import it. Reduce requests from server. Only import custom font here or we can import icons in less file like above app/design/frontend/Magento/blank/web/css/source/_icons.less